app/controllers/orders_controller.rb

```diff
@@ -8,7 +8,7 @@ def new
   end
 
   def show
-    @order = Order.find(params[:id])
+    @order = Order.find(params[:id] || params[:order_id])
```

Comment:
I'm guessing you needed this because we had some inconsistencies with how that id was passed into params?

I think ideally we would want to edit our routes to keep it just as :id but, this is a really cool solution you found to this! 

No need to change.

Reply:
I think totally ideally we would want to keep things the same.  I think that because I had two different types of params coming through, this was the best way for me to be able to ensure that other params could enter while delivering on the same functionality.

But now that you mention it, maybe this was a sign that a new controller needed to be built, but I didn't build it?  Something to reflect on moving forward.